app/views/plate_creation/donor_pooling_plate.html.erb

```diff
@@ -0,0 +1,44 @@
+<%= page('donor-pooling-page') do %>
+  <%= content do %>
+    <%= card id: 'main-content' do %>
+      <div>
+        <h3>Instructions</h3>
+        <p>This step uses multiple source plates for pooling together into a destination plate.</p>
+        <p>This process checks that the source plates are compatible and can be used together.</p>
```

Comment:
Wonder if users might read this and think 'what would make 2 plates incompatible?' 

That's what I thought. Do you mean it checks the constraints - study, project, donor etc? Or that they are of the correct purpose?

Reply:
Updated the Instructions with the conditions for pooling. It should give an idea about what happened if pooling fails.